velox/functions/sparksql/tests/DateTimeFunctionsTest.cpp

```diff
@@ -898,6 +898,15 @@ TEST_F(DateTimeFunctionsTest, fromUnixtime) {
 
   // 8 hours ahead UTC.
   setQueryTimeZone("Asia/Shanghai");
+// In debug mode, Timestamp constructor will throw exception if range check
+// fails.
+#ifdef NDEBUG
+  // Integer overflow in the internal conversion from seconds to milliseconds.
+  EXPECT_EQ(
+      fromUnixTime(std::numeric_limits<int64_t>::max(), "yyyy-MM-dd HH:mm:ss"),
```

Comment:
The example in the documentation shows a different result: https://facebookincubator.github.io/velox/functions/spark/datetime.html#from_unixtime

```
SELECT from_unixtime(9223372036854775807, "yyyy-MM-dd HH:mm:ss");  -- '1969-12-31 23:59:59'
```

Which one is the "right" one?

Reply:
Both are right. The example in the documentation doesn't set query time zone configuration, but this test sets time zone to "Asia/Shanghai" , so the results are different.

Reply:
Maybe we should add another example in document for query with time zone.

Reply:
I added example in document : 59968ef17da11b45830241d3df1dc93e8d89f900